regression/scripts/motor/motor_test.py

```diff
@@ -0,0 +1,172 @@
+# motor_test.py
```

Comment:
Actually on the contrary. The motor script was one of the least covered scripts. When I first made this regression there were a lot of bugs that were initially not caught because the functions were never called